diff --git a/node_modules/zod-prisma-types/dist/utils/getPackageVersion.js b/node_modules/zod-prisma-types/dist/utils/getPackageVersion.js
index bb21b61..d7fe6ca 100644
--- a/node_modules/zod-prisma-types/dist/utils/getPackageVersion.js
+++ b/node_modules/zod-prisma-types/dist/utils/getPackageVersion.js
@@ -16,9 +16,9 @@ const getPackageVersion = (packageName, options) => {
     try {
         const rawData = fs_1.default.readFileSync(path_1.default.join(process.cwd(), 'package.json'), 'utf-8');
         const parsedJsonData = zod_1.z
-            .object({ dependencies: zod_1.z.record(zod_1.z.string(), zod_1.z.string()) })
+            .object({ dependencies: zod_1.z.record(zod_1.z.string(), zod_1.z.string()).optional(), devDependencies: zod_1.z.record(zod_1.z.string(), zod_1.z.string()).optional() })
             .parse(JSON.parse(rawData));
-        let version = parsedJsonData.dependencies[packageName];
+        let version = (parsedJsonData.dependencies || {})[packageName] || (parsedJsonData.devDependencies || {})[packageName];
         if (!version) {
             if (options === null || options === void 0 ? void 0 : options.fallbackVersion) {
                 console.log('\x1b[33m', `[WARNING] Falling back to default ${options.fallbackVersion.major}.${options.fallbackVersion.minor}.${options.fallbackVersion.patch} because the version of the package ${packageName} could not be determined - make sure it is installed as a dependency and not a devDependency`, '\x1b[37m');
